Microsoft.Toolkit.Uwp.UI/Extensions/Tree/LogicalTree.cs

```diff
@@ -314,11 +691,29 @@ public static object TryFindResource(this FrameworkElement start, object resourc
 
                 current = current.Parent as FrameworkElement;
             }
+            while (current is not null);
 
-            // Finally try application resources.
-            Application.Current?.Resources?.TryGetValue(resourceKey, out value);
+            // Finally try application resources
+            _ = Application.Current?.Resources?.TryGetValue(resourceKey, out value);
 
             return value;
         }
+
+        /// <summary>
+        /// Provides a WPF compatible version of TryFindResource to provide a static resource lookup.
```

Comment:
This one is new, so should we modify this comment to not call out `WPF compatible`?

Reply:
I was thinking the "compatible" bit here refers mostly to how the resource lookup logic itself is implemented (as in, hierarchical starting from the current element and going up, falling back to the global application resources) rather than the exact signature? So with the core semantics being the same, but with this overload just offering a slightly different convenient helper around it? Happy to change this summary to something else if you prefer though 🙂

Reply:
Sound fine, no worries. Just wanted to check and get thoughts.